tasks/build-preview-urls-comment.cjs

```diff
@@ -16,12 +16,12 @@ const slugify = require('@sindresorhus/slugify');
 const buildPreviewURLComment = (ref) => {
     const branch = ref.replace('refs/heads/', '');
     const branchSlug = slugify(branch);
-    const comment = `# Branch Preview URLs
+    const comment = `# Branch Preview
```

Comment:
This title is clearly bound in a very fragile way into ensuring the posted comment is _updated_ rather than duplicated, here: 
https://github.com/adobe/spectrum-web-components/blob/7cdc962124102b714d04dab59691daff96d87f9b/.github/workflows/test.yml#L41

Reply:
Ah-ha!  Good catch.  If you like the change, I can update there too or revert this one. My thought was, the links below speak for themselves as being URLs.  Might also let us expand to including snapshots if we wanted to 🤷